handbook/people-ops/onboarding/general_onboarding.md

```diff
@@ -6,7 +6,7 @@
 
 - **[Google Workspace/Gmail](https://www.google.com/gmail/)**: Access all Google Workspace apps in our org (Gmail, Google Docs, Google Sheets, Google Calendar).
 - **[Greenhouse](https://www.greenhouse.io/):** Access our recruiting tool for tracking candidates in the hiring pipeline.
-- **[Slack](https://slack.com/)**: Access our team communication tool.
+- **[Slack](https://slack.com/)**: Access our team communication tool. For consistency, use First Name (or preferred name)+Last Name as your display name on Slack.
```

Comment:
```suggestion
- **[Slack](https://slack.com/)**: Access our team communication tool. For consistency, use `First Name (or preferred name) Last Name` as your display name on Slack.
```

Instead of a "+" here, I think it might be more clear to put it in a code block like this